bulma/templates/bulma/pagination.html

```diff
@@ -0,0 +1,42 @@
+{% load bulma_tags %}
+{% with bpurl=bulma_pagination_url|default:"" %}
```

Comment:
We already have pagination, please see here: https://github.com/timonweb/django-bulma/blob/master/bulma/templates/pagination.html

Reply:
how to use it ? 
```
{% include pagination.html  with page_obj=xxxx %}

```